_travis.yml.backup

```diff
@@ -6,7 +6,6 @@ env:
         - PLAT=x86_64
         - NP_BUILD_DEP="numpy==1.16.5"
         - CYTHON_BUILD_DEP="Cython==0.29.18"
-        - PYBIND11_BUILD_DEP="pybind11==2.4.3"
```

Comment:
I think this is still used by a line that hasn't been deleted?

Reply:
thanks. I reverted this change and changed the `NP_*` requirement use `oldest-supported-numpy` instead, like on azure.